docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -111,11 +112,11 @@ If all went to plan, you'll see this:
 +-----------------------------------------------------+----------------+--------------+-----------------+
 ```
 
-Glorious. You have the `10000` TNT to stake one validator, and a bit extra to cover the transaction fees. Now it's time to get your Validator running.
+Glorious. You have the `10000 HNT` required to stake one validator, and a bit extra to cover the transaction fees. Now it's time to get your Validator running.
```

Comment:
This talks about having extra to cover transaction fees but I don't see where sending more than 10,000 HNT or how much more is advised or clarified...